wiki/Game_modifier/Hidden/id.md

```diff
@@ -0,0 +1,81 @@
+---
+stub: true
+tags:
+  - hidden
+  - HD
+  - mod
+  - game modifier
+---
+
+# Hidden (mod)
+
+![HD mod icon](/wiki/shared/mods/HD.png "Ikon mod Hidden (HD) mod")
+
+*Untuk melihat daftar keseluruhan dari semua mod, kunjungi: [Game Modifier](/wiki/Game_modifier)*\
+*Berbeda konteks dengan [Fade In (mod)](/wiki/Game_modifier/Fade_In) ataupun [Flashlight (mod)](/wiki/Game_modifier/Flashlight).*
+
+## Tentang
+
+- Singkatan: HD
+- Tipe: Peningkat Kesulitan
+- Pengali skor:
+  - ![][o!s] ![][o!t] ![][o!c]: 1.06x
+  - ![][o!m]: 1.00x
+- Tombol shortcut default: `F`
+  - Tombol shortcut default ([osu!mania](/wiki/Game_mode/osu!catch)): `F` `F` atau `Shift`+`F`
+- Caption:
+  - ![][o!s]: `Approach circle serta not-not akan memudar/menghilang perlahan-lahan, namun nilai maksimal Anda akan sedikit bertambah.`
+  - ![][o!t]: `Not-not akan menghilang perlahan sebelum anda mengenainya!`
```

Comment:
```suggestion
  - ![][o!t]: `Not-not akan menghilang perlahan sebelum Anda mengenainya!`
```

Reply:
this follows the caption that is available via the in-game translation, so changing them might not be a good idea..